Cargo.toml

```diff
@@ -70,6 +70,7 @@ fundu = "2.0.1"
 futures = "0.3.30"
 globset = "0.4.15"
 graph-rs-sdk = { git = "https://github.com/spiceai/graph-rs-sdk", rev = "f8703df260146b313461029d41c4a021306832b8" }
+humantime = "2.1.0"
```

Comment:
We already use `fundu` for this purpose - I'd prefer to reuse that if possible

Reply:
Oh I see we're already using `humantime` for something else - we should just pick one.

Reply:
I haven't removed `humantime` in this PR, but I've changed to using `fundu` instead because it seems to have more usage in the repo.